src/lib/nvme_pt_ioctl.c

```diff
@@ -413,8 +413,10 @@ int opal_init_pt(struct sed_device *dev, const char *device_path)
 	return ret;
 }
 
-int opal_level0_discv_info_pt(struct sed_device *dev,
-				struct sed_opal_level0_discovery *discv)
+static int opal_tper_host_prep(struct sed_device *dev);
```

Comment:
prefer this declaration to move to the top of the file.

Reply:
will do it